analyzers/tests/SonarAnalyzer.Test/TestCases/SymbolicExecution/Roslyn/ConditionEvaluatesToConstant.cs

```diff
@@ -3964,8 +3964,7 @@ public void Method()
         var success = false;
         Exception exception = null;
         int retries = 3;
-        while (!success && retries > 0) // Noncompliant FP
-        //                 ^^^^^^^^^^^
+        while (!success && retries > 0) // Compliant
```

Comment:
In the ITs, there was a modifield file that moved issue from `End While` to the following `If`. What about documenting that test case (minimalized) here?